proxy/src/http/source.rs

```diff
@@ -713,42 +714,48 @@ mod test {
 
     use crate::avatar;
     use crate::coco;
+    use crate::error;
     use crate::identity;
 
-    #[tokio::test]
-    async fn blob() {
-        let tmp_dir = tempfile::tempdir().unwrap();
-        let librad_paths = Paths::from_root(tmp_dir.path()).unwrap();
-        let (platinum_id, _platinum_project) = coco::replicate_platinum(
-            &tmp_dir,
-            &librad_paths,
-            "git-platinum",
-            "fixture data",
-            "master",
+    fn assert_status(res: &Response<Bytes>) {
```

Comment:
It would be nice to have a `http::test::assert_response` which takes the expected `StatusCode` and deserialised response value, errors early if the statuscode doesn't match. Pretty much extracting this pattern:

``` rust
let have: Value = serde_json::from_slice(res.body()).unwrap();

assert_eq!(res.status(), StatusCode::OK);
assert_eq!(
    have,
    json!(registry::Org {
        id: org_id.clone(),
        shareable_entity_identifier: format!("%{}", org_id.to_string()),
        avatar_fallback: avatar::Avatar::from(&org_id.to_string(), avatar::Usage::Org),
        members: vec![user]
    })
);
```